java/HealthVault Mobile/HVSample/src/main/res/values/dimens.xml

```diff
@@ -81,4 +81,50 @@
 	<dimen name="profileimagelayout_marginleftt">142dp</dimen>
 	<dimen name="profileimagelayout_marginbottom">28dp</dimen>
 
+	<!--Medications -->
+	<dimen name="medication_main_marginleft">20dp</dimen>
+	<dimen name="medication_main_marginright">125dp</dimen>
+	<dimen name="medication_main_margintop">22dp</dimen>
+	<dimen name="medication_main_height">24dp</dimen>
+	<dimen name="medication_main_width">215dp</dimen>
+	<dimen name="medication_list_width">360dp</dimen>
+	<dimen name="medication_list_height">88dp</dimen>
+	<dimen name="medication_textsize">24sp</dimen>
+	<!--Medications record-->
+	<dimen name="medicationname_marginleft">20dp</dimen>
```

Comment:
To follow up on previous comments, we should change all these names to follow a consistent pattern. If we are using _ then it should go between each word.